<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcesGui</class>
 <widget class="QWidget" name="ResourcesGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget_resources">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="buttonsPage">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>341</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_resources">
        <item>
         <widget class="QPushButton" name="pushButton_profileUpdate">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>Update Profile Photo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_changeColor">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>Change Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_createMessage">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>Create Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_viewMessage">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>View Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteMessage">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>Delete Message</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>491</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;.AppleSystemUIFont&quot;;</string>
       </property>
       <property name="text">
        <string>Costumize your exprience with the following resources</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="createMessagePage">
      <widget class="QPushButton" name="pushButton_saveMessage">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>480</y>
         <width>186</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
       </property>
       <property name="text">
        <string>Save Message</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_title">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>155</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_text">
       <property name="geometry">
        <rect>
         <x>96</x>
         <y>213</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Text:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(220,20,60);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="lineEdit_text">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>311</width>
         <height>231</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>311</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;.AppleSystemUIFont&quot;;</string>
       </property>
       <property name="text">
        <string>Write and save your message here</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="viewMessagePage">
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>341</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_view">
        <item>
         <widget class="QLabel" name="label_titleView">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Title:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_choices"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_view">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(220,20,60);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_textOutput">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>331</width>
         <height>151</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>281</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;.AppleSystemUIFont&quot;;</string>
       </property>
       <property name="text">
        <string>Read your saved messages</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="deleteMessagePage">
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>143</x>
         <y>240</y>
         <width>411</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_deleteMessage">
        <item>
         <widget class="QLabel" name="label_titleDeleteMessage">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Title:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_choicesDelete"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_back3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(220,20,60);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>281</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;.AppleSystemUIFont&quot;;</string>
       </property>
       <property name="text">
        <string>Delete undesriable message</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_editProfile">
      <widget class="QPushButton" name="pushButton_picture">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>260</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Uptade Picture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_image">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>141</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Update Name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
